static/css/dev_menu.scss

```diff
@@ -24,4 +24,17 @@
 
 #chart-region {
   margin-left: 400px;
+}
+
+#drill .noborder{
```

Comment:
Are these two classes used to remove the extra line?

Reply:
Yes.